package.json

```diff
@@ -28,5 +28,8 @@
   "homepage": "https://github.com/CliffCloud/Leaflet.EasyButton",
   "dependencies": {
     "leaflet": "^1.0.1"
+  },
+  "devDependencies": {
+    "@types/leaflet": "^1.2.0"
```

Comment:
This makes it so the types are installed when running `npm i` so people developing against the library do not have any errors in the type file.